### PR TITLE
nsqadmin: connections count fixes

### PR DESCRIPTION
**Pull request by ploxiln**
The total connections count on the Topic page was accidentally broken recently in #1118.

I'm not sure when the rate column in the Totals row of the Channel page started having an issue, the code has been that way since 2015 ...

I also had some problems with gulp:

with nodejs 10.x:

```
$ ./gulp clean build
internal/util/inspect.js:31
const types = internalBinding('types');
              ^

ReferenceError: internalBinding is not defined
    at internal/util/inspect.js:31:15
    at req_ (/Users/pierce/go/src/github.com/nsqio/nsq/nsqadmin/node_modules/natives/index.js:137:5)
    at require (/Users/pierce/go/src/github.com/nsqio/nsq/nsqadmin/node_modules/natives/index.js:110:12)
    at util.js:25:21
    at req_ (/Users/pierce/go/src/github.com/nsqio/nsq/nsqadmin/node_modules/natives/index.js:137:5)
    at require (/Users/pierce/go/src/github.com/nsqio/nsq/nsqadmin/node_modules/natives/index.js:110:12)
    at fs.js:42:21
    at req_ (/Users/pierce/go/src/github.com/nsqio/nsq/nsqadmin/node_modules/natives/index.js:137:5)
    at Object.req [as require] (/Users/pierce/go/src/github.com/nsqio/nsq/nsqadmin/node_modules/natives/index.js:54:10)
    at Object.<anonymous> (/Users/pierce/go/src/github.com/nsqio/nsq/nsqadmin/node_modules/graceful-fs/fs.js:1:99)
```

first attempt with nodejs 8.x:

```
$ ./gulp clean build
[16:54:15] Using gulpfile ~/go/src/github.com/nsqio/nsq/nsqadmin/gulpfile.js
[16:54:15] Starting 'clean'...
[16:54:15] Finished 'clean' after 6.57 ms
[16:54:15] Starting 'sass'...
[16:54:15] Finished 'sass' after 2.8 ms
[16:54:15] Starting 'browserify'...
[16:54:15] Starting 'sync-static-assets'...
[16:54:15] Starting 'vendor-build-js'...
[16:54:15] 'sync-static-assets' errored after 61 ms
[16:54:15] Error: EEXIST: file already exists, mkdir '/Users/pierce/go/src/github.com/nsqio/nsq/nsqadmin/static/build'
events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open '/Users/pierce/go/src/github.com/nsqio/nsq/nsqadmin/static/build/base.css'
Segmentation fault: 11
```

just doing a "gulp clean" on its own seemed to resolve all my issues after that.

cc @mreiferson @jehiah

### CODE REVIEW

**Comment by ploxiln** [comment]
a bit more reference, from https://github.com/gulpjs/gulp-cli#flags

> --series |   | Run tasks given on the CLI in series (the default is parallel).

File: nsqadmin/README.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ Read the [docs](http://nsq.io/components/nsqadmin.html)
 
 ## Working Locally
 
+ 0. install nodejs 8.x (includes npm)
  1. `$ npm install`
- 2. `$ ./gulp clean watch` or `$ ./gulp clean build`
+ 2. `$ gulp --series clean watch` or `$ ./gulp --series clean build`
```
Comment (mreiferson) [inline nsqadmin/README.md]: first `gulp` command needs the `./` prefix

**Comment by mreiferson** [comment]
LGTM otherwise

File: nsqadmin/README.md (hunk 1 above)
Comment (ploxiln) [reply nsqadmin/README.md]: thanks, squashed in that tiny fix

**Comment by ploxiln** [comment]
By the way, I think the gulp issue I ran into was not because `clean` and `build` were run in parallel (that might be new in gulp 4.x and the `--series` flag is also new), I think it was because of `sync-static-assets`,`vendor-build-js`, and `browserify` running in parallel (https://github.com/gulpjs/gulp/blob/v3.9.1/docs/API.md#gulptaskname--deps-fn) and while I think they should be independent, they can all try to "mkdir build" if it is missing.

**Pull request merged**